extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/SneakyThrow.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.narayana.jta.runtime;
+
+/**
+ * An utility class which makes possible to throw any exception as a {@link RuntimeException}.
+ * It means to throw checked exception (subtype of Throwable or Exception) as un-checked exception.
+ * This considers the Java 8 inference rule that states that a {@code throws E} is inferred as {@code RuntimeException}.
+ */
+public class SneakyThrow {
```

Comment:
Considering that this class is used only in this package, can you make it package-protected?

Reply:
Or even better, move the utility method to the `TransactionalInterceptorBase` class as a private method

Reply:
sure, done